blueprints/filestartercontent/blueprint.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
I see other blueprints directories are in the form `this-that-another`. Maybe we should name this `file-starter-content`? What do you think?

Reply:
Good idea, keeping consitency.